lynx/common/player.py

```diff
@@ -10,10 +13,68 @@ class Player(Serializable):
     player_resources: Dict[str, int] = field(default_factory=dict)
     drop_area: Vector = field(default_factory=Vector)
     agents: dict[str, Agent] = field(default_factory=dict)
+    max_agents: int = 1
```

Comment:
```
max_agents: int = 1

BASE_COSTS = {
    "Wood": 6,
    "Stone": 6,
}
```
[Optional] Move these values to a config (just like [here](https://github.com/group-project-gut/lynx-scene-host-python/blob/master/config.ini)) as e.g. `START_MAX_AGENTS`, `WOOD_COST`, `STONE_COST`